elasticsearch/files/elasticsearch.yml

```diff
@@ -41,7 +41,9 @@ cluster.name: {{ server.cluster.name }}
 # from configuring them manually. You can tie this node to a specific name:
 #
 # node.name: "Franz Kafka"
-node.name: {{ server.get('name', '${HOSTNAME}') }}
+{%- if server.name is defined %}
+node.name: {{ server.name }}
```

Comment:
Yes, and why is that wrong? You can use this notation, see eg.
https://discuss.elastic.co/t/can-you-set-es-node-name-to-same-as-the-machine-hostname/900